src/lib/netlist/devices/nld_roms.cpp

```diff
@@ -96,6 +96,59 @@ namespace netlist
 			nld_power_pins m_power_pins;
 		};
 
+		NETLIB_OBJECT(mcm14524_rom)
+		{
+			NETLIB_CONSTRUCTOR(mcm14524_rom)
+			, m_enabled(*this, "m_enabled", true)
+			, m_latched_rom(*this, "m_latched_rom", 0)
+			, m_A(*this, 1, "A{}", NETLIB_DELEGATE(addr))
+			, m_CLK(*this, "CLK", NETLIB_DELEGATE(addr))
+			, m_clk_old(*this, "m_clk_old", true)
+			, m_EN(*this, "EN", NETLIB_DELEGATE(en))
+			, m_B(*this, 1, "B{}", 0)
+			, m_ROM(*this, "ROM")
+			, m_power_pins(*this)
+			{
+			}
+
+		private:
+			inline NETLIB_HANDLERI(en)
+			{
+				m_enabled = m_EN();
+				uint8_t o = m_enabled ? m_latched_rom : 0; // outputs are forced to 0 by enable going low; this chip does not have tri-state outputs!
+				for (std::size_t i=0; i<4; i++)
+				{
+					m_B[i].push((o >> i) & 1, NLTIME_FROM_NS(137)); // the TaccEn time is dependent on (roughly) 1150/<input voltage>, this value is hard-coded for 8.4VDC
+				}
+			}
+
+			inline NETLIB_HANDLERI(addr)
+			{
+				if (!m_CLK() && m_clk_old) // latch on falling edge
+				{
+					const auto addr = m_A();
+					m_latched_rom = m_ROM[addr];
+				}
+				m_clk_old = m_CLK();
+				uint8_t o = m_enabled ? m_latched_rom : 0; // outputs are forced to 0 by enable going low; this chip does not have tri-state outputs!
+				for (std::size_t i=0; i<4; i++)
+				{
+					m_B[i].push((o >> i) & 1, NLTIME_FROM_NS(702)); // interpolated from TaccC on the datasheet for the 8.4VDC case: ns*voltage is 6750 for 5v, 5500 for 10v, interpolated to 5900 for 8.4v, and 5900/8.4 = 702.361ns
```

Comment:
Please see the CD4006 for an example how to implement supply voltage depending timing.

Reply:
This is now done for the MCM14524